src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -221,6 +284,16 @@ def run_conversion(
             If True, only a subset of the data will be added for testing purposes, by default False.
 
         """
+        # Deprecate external_mode parameter
+        # TODO: Remove after May 2026
+        if external_mode is not True:
+            warnings.warn(
+                "The 'external_mode' parameter is deprecated and will be removed in May 2026. "
```

Comment:
On or after May 2026, only external videos will be supported by the LightningPoseConverter.